kdestandardize.ado

```diff
@@ -234,6 +234,34 @@ prog def kdestandardize
 				   9 `"INFORMATION SUPPORT AND SERVICES"'		 			 ///
 				   10 `"TECHNOLOGY"', modify
 				   
+	   // Define PERFORMANCE_TYPE
+	   la def ptype 1 `"Points"'											 ///
+				    2 `"NAPD Calculation"', modify
+					
+	   // Define ASSESSMENT_LEVEL
+	   la def assesslvl 1 `"Kentucky"'										 ///
+						2 `"Nation"', modify
+						
+		// Define PERFORMANCE_MEASURE
+	   la def pmsr 1 `"Academic Attainment - Mathematics - 1S2"'       		 ///
+				   2 `"Academic Attainment - Reading - 1S1"'				 ///
+				   3 `"Non-Traditional Completion - 6S2"'	 			     ///
+				   4 `"Non-Traditional Participation - 6S1"'				 ///
+				   5 `"Secondary Placement - 5S1"'							 ///
+				   6 `"Secondary School Completion - 3S1"'					  ///
+				   7 `"Student Graduation Rate - 4S1"'						 ///
+				   8 `"Technical Skill Attainment - 2S1"', modify
+				   
+	   // Define COHORT_TYPE
```

Comment:
Haven't made it to this one just yet, but this is a place where reshaping the data would make it easier to work with.  It would require more columns to store things, but then the columns would identify the 4/5year cohort values.